.gitignore

```diff
@@ -1,5 +1,7 @@
 .tox
 .hg*
+.venv*
+.idea
```

Comment:
I believe these two lines should not be included because they're specific to one setup. It would make this file harder to maintain over time, for little benefit :-)